src/components/posts/PostForm.jsx

```diff
@@ -1,7 +1,18 @@
 /** @jsx createVNode */
 import { createVNode } from "../../lib";
+import { globalStore } from "../../stores";
 
 export const PostForm = () => {
+  const { addPost } = globalStore.actions;
+  const handleClick = () => {
+    const content = document.getElementById("post-content").value;
```

Comment:
DOM 요소를 직접 조작하기 보다 상태 관리를 통해 폼을 제어해보는건 어떨까용??
예를들면, 상태 초기화와 함께 DOM 업데이트를 하는 방법이 있을 것 같은데 요렇게 한번 제안해 보겠습니다!

```suggestion
const handleClick = () => {
    if (!content.trim()) {
      alert("내용을 입력해주세요");
      return;
    }
    addPost(content);

    content = '';
    document.getElementById("post-content").value = "";
  };
```